plugins/modules/ipaconfig.py

```diff
@@ -175,7 +175,7 @@
     enable_sid:
         description: >
           New users and groups automatically get a SID assigned.
-          Requires IPA 4.9.8+.
+          Cannot be deactivated once activate. Requires IPA 4.9.8+.
```

Comment:
I think this should be "once active" or "once activated", right?

Reply:
fixed.